src/Spryker/Zed/RabbitMq/Business/Model/Admin/AdminPermissionHandler.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Spryker\Zed\RabbitMq\Business\Model\Admin;
+
+use Psr\Log\LoggerInterface;
+use Spryker\Zed\RabbitMq\Dependency\Guzzle\RabbitMqToGuzzleInterface;
+
+class AdminPermissionHandler implements AdminPermissionHandlerInterface
+{
+    /**
+     * @var \Spryker\Zed\RabbitMq\Dependency\Guzzle\RabbitMqToGuzzleInterface
+     */
+    protected $client;
+
+    /**
+     * @var string
+     */
+    protected $apiAdminUrl;
+
+    /**
+     * @var string
+     */
+    protected $username;
+
+    /**
+     * @var string
+     */
+    protected $password;
+
+    /**
+     * @param \Spryker\Zed\RabbitMq\Dependency\Guzzle\RabbitMqToGuzzleInterface $client
+     * @param string $apiAdminUrl
+     * @param string $username
+     * @param string $password
+     */
+    public function __construct(RabbitMqToGuzzleInterface $client, $apiAdminUrl, $username, $password)
+    {
+        $this->client = $client;
+        $this->apiAdminUrl = $apiAdminUrl;
+        $this->username = $username;
+        $this->password = $password;
+    }
+
+    /**
+     * @param \Psr\Log\LoggerInterface $logger
+     *
+     * @return bool
+     */
+    public function setPermissions(LoggerInterface $logger)
+    {
+        $response = $this->client->put($this->apiAdminUrl, [
+            'auth' => [$this->username, $this->password],
+            'json' => [
+                'configure' => '.*',
+                'read' => '.*',
+                'write' => '.*',
+            ],
+        ]);
+
+        if ($response->getStatusCode() > 200 && $response->getStatusCode() < 300) {
```

Comment:
is there a better way to confirm the permission has been added?

Reply:
The response body is empty